src/Val.cc

```diff
@@ -938,10 +949,39 @@ static std::variant<ValPtr, std::string> BuildVal(const rapidjson::Value& j, con
         }
 
         case TYPE_INTERVAL: {
-            if ( ! j.IsNumber() )
-                return mismatch_err();
+            if ( j.IsNumber() )
+                return make_intrusive<IntervalVal>(j.GetDouble());
+
+            if ( j.IsString() ) {
+                auto parts = util::split(j.GetString(), " ");
```

Comment:
I'm not sure we care all that much, but IIRC superfluous spaces would yield empty parts. Could filter out empty parts to support more than one spaces as separators. Mostly relevant if someone crafts input by hand.